### PR TITLE
Feat/UI changes

### PR DESCRIPTION
**Pull request by manish591**
### Changes Made In This PR

* Added Home page, here user can see a landing page with some buttons
* Added some style changes, like added hover state to improve interactivity

### CODE REVIEW

File: src/components/navbar/Navbar.css [hunk 1]
```diff
@@ -16,12 +16,17 @@
 
 .navbar__search-group {
   align-items: center;
-  padding-block: 0.8rem;
+  padding-block: 1rem;
   padding-inline: 0.4rem;
   border: 1px solid black;
```
Comment (OmKakatkar) [inline src/components/navbar/Navbar.css:21]: For consistency use variables for colors

**Review by OmKakatkar — APPROVED** [review]
Looks good to merge:

I really liked:
* Folder Structure, the pages, components and assets are separated.
* A single export file for the Pages folder.
*  Consistency of variable names.

**Review by romabulani — APPROVED** [review]
- I liked that you have implemented all the routes inside `explore`, will also implement in my project
- Good folder structure, indentation and readability
- Good to merge

Suggestion:
- If possible, add preview link in PR as it becomes easy to review the code

**Comment by manish591** [comment]
> * I liked that you have implemented all the routes inside `explore`, will also implement in my project
> * Good folder structure, indentation and readability
> * Good to merge
> 
> Suggestion:
> 
> * If possible, add preview link in PR as it becomes easy to review the code

Will do it from next time

**Pull request merged**